core/test/usTestDriverActivator.cpp

```diff
@@ -57,6 +57,7 @@ US_IMPORT_BUNDLE(CppMicroServices)
 US_IMPORT_BUNDLE(TestBundleA)
 US_IMPORT_BUNDLE(TestBundleA2)
 US_IMPORT_BUNDLE(TestBundleB)
+US_IMPORT_BUNDLE(TestBundleLQ_1)
```

Comment:
Shouldn't all the other "TestBundleLQ_" bundles be listed here as statically linked bundles as well?